app_dart/lib/src/request_handlers/get_benchmarks.dart

```diff
@@ -30,26 +33,102 @@ class GetBenchmarks extends RequestHandler<Body> {
 
   @override
   Future<Body> get() async {
-    final String branch = request.uri.queryParameters[branchParam] ?? 'master';
-    const int maxRecords = 50;
+    const String master = 'master';
+
+    final int maxRecords = config.maxRecords;
+    final String branch = request.uri.queryParameters[branchParam] ?? master;
     final DatastoreService datastore = datastoreProvider(config.db);
-    final DatastoreDB db = datastore.db;
+    final List<Map<String, dynamic>> benchmarks = <Map<String, dynamic>>[];
+
+    Map<String, Result> releaseBranchMap = <String, Result>{};
+    Map<String, Result> masterMap = <String, Result>{};
+    int masterLimit = maxRecords;
```

Comment:
rm and use the config.maxRecords.

Reply:
Replace maxRecords with config.maxRecords. But we need to keep variable masterLimit, which is used to track the number of `master` commits.